Classes/Middleware/TokenRefreshMiddleware.php

```diff
@@ -86,7 +87,7 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
                 if (preg_match_all($pattern, $content, $foundJwtTokens)) {
                     foreach ($foundJwtTokens[1] as $foundJwtToken) {
                         try {
-                            $data = JWT::decode($foundJwtToken, $GLOBALS['TYPO3_CONF_VARS']['SYS']['encryptionKey'], ['HS256']);
+                            $data = JWT::decode($foundJwtToken, new Key($GLOBALS['TYPO3_CONF_VARS']['SYS']['encryptionKey'], 'HS256'));
                             if ((int)$data->user !== $currentUserId) {
                                 $data->user = $currentUserId;
                                 $newToken = JWT::encode($data, $GLOBALS['TYPO3_CONF_VARS']['SYS']['encryptionKey'], 'HS256');
```

Comment:
@jpmschuler Could you also add a type cast `(array)` to the `$data` argument in line 93?
```suggestion
                                $newToken = JWT::encode((array)$data, $GLOBALS['TYPO3_CONF_VARS']['SYS']['encryptionKey'], 'HS256');
```

Reply:
done